frontend/src/components/editor-page/sidebar/users-online-sidebar-menu/users-online-sidebar-menu.tsx

```diff
@@ -33,28 +34,35 @@ export const UsersOnlineSidebarMenu: React.FC<SpecificSidebarMenuProps> = ({
   const buttonRef = useRef<HTMLButtonElement>(null)
   const realtimeUsers = useApplicationState((state) => state.realtimeStatus.onlineUsers)
   useTranslation()
+  const ownUsername = useApplicationState((state) => Optional.ofNullable(state.user?.username).orElseGet(() => ''))
+  const ownDisplayname = useApplicationState((state) => state.realtimeStatus.ownUser.displayName)
+  const ownStyleIndex = useApplicationState((state) => state.realtimeStatus.ownUser.styleIndex)
 
   useEffect(() => {
-    buttonRef.current?.style.setProperty('--users-online', `"${realtimeUsers.length}"`)
+    buttonRef.current?.style.setProperty('--users-online', `"${realtimeUsers.length + 1}"`)
   }, [realtimeUsers])
 
   const hide = selectedMenuId !== DocumentSidebarMenuSelection.NONE && selectedMenuId !== menuId
   const expand = selectedMenuId === menuId
   const onClickHandler = useCallback(() => onClick(menuId), [menuId, onClick])
 
+  // ToDo: use real color
+  const ownUserElement = useMemo(
```

Comment:
Memo not necessary here. tbh i would rather extract this and the used hooks into a new component to reduce the complexity of this file.